src/input/input_manager.h

```diff
@@ -0,0 +1,135 @@
+//
+// Created by grzegorz on 17.12.18.
+//
+
+#pragma once
+
+#include "GLFW/glfw3.h"
+
+#define MAX_KEYS 1024
```

Comment:
Do we really have 1024 keys? Maybe there is some possibility to make it somewhat generic?